lib/dotcom/system_status/groups.ex

```diff
@@ -0,0 +1,221 @@
+defmodule Dotcom.SystemStatus.Groups do
+  @moduledoc """
+  A module that groups alerts into statuses for the system status widget.
+  """
+
+  alias Alerts.Alert
+
+  @lines ["Blue", "Orange", "Red", "Green"]
+  @green_line_branches ["Green-B", "Green-C", "Green-D", "Green-E"]
+  @routes ["Blue", "Orange", "Red", "Mattapan"] ++ @green_line_branches
+
+  def groups(alerts, now) do
```

Comment:
I don't *think* that `now` actually represents now. It should be `date_time` unless it really is `now` in which case it shouldn't be an argument.